pkg/orchestrator/config/config.go

```diff
@@ -165,7 +165,8 @@ func NewOrchestratorForwarder(log log.Component) forwarder.Forwarder {
 	if !config.Datadog.GetBool(key(orchestratorNS, "enabled")) {
 		return nil
 	}
-	if flavor.GetFlavor() == flavor.DefaultAgent && !config.IsCLCRunner() {
+	fl := flavor.GetFlavor()
+	if (fl == flavor.DefaultAgent || fl == flavor.ProcessAgent) && !config.IsCLCRunner() {
```

Comment:
Actually from the above comment I don't think we need to modify this file to check if the flavor can be the process agent as this code path should only be used in the core agent / CLC runner 🤔

Reply:
Ah thank you! I will revert this change 🙇🏻‍♀️